src/utils/constants.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Delete this file since it is unused. If ever we do end up using it in the future, we can create it again.